lizmap/modules/lizmap/install/upgrade_presentation.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class lizmapModuleUpgrader_userfields extends jInstallerModule
+{
+    public $targetVersions = array(
+        '3.8pre',
```

Comment:
You must indicate here only versions of other branches if it is backported in previous branches, and the next version of the current branch . so here only `['3.8.0-alpha.1']` or `['3.8.0-pre.20240308']`, so you must put at the same time this version into module.xml.